api/android/api/src/main/jni/Android.mk

```diff
@@ -62,9 +68,20 @@ endif
 # external libs and sub-plugins
 #------------------------------------------------------
 ifeq ($(ENABLE_TF_LITE),true)
-NNS_API_FLAGS += -DENABLE_TENSORFLOW_LITE=1
+NNS_API_FLAGS += \
+    -DENABLE_TENSORFLOW_LITE=1 \
+    -DTFLITE_VERSION=$(TFLITE_VERSION) \
+    -DTFLITE_VERSION_MAJOR=$(TFLITE_VERSION_MAJOR) \
+    -DTFLITE_VERSION_MINOR=$(TFLITE_VERSION_MINOR) \
+    -DTFLITE_VERSION_MICRO=$(TFLITE_VERSION_MICRO)
+
 # define types in tensorflow-lite sub-plugin. This assumes tensorflow-lite >= 1.13 (older versions don't have INT8/INT16)
 NNS_API_FLAGS += -DTFLITE_INT8=1 -DTFLITE_INT16=1
```

Comment:
let's consider tf-lite ver here (tf-lite > 1.13)

Reply:
Updated! And move this part to `Android-tensorflow-lite-.mk`